src/test/kotlin/org/elm/ide/lineMarkers/ElmRecursiveCallLineMarkerProviderTest.kt

```diff
@@ -0,0 +1,22 @@
+package org.elm.ide.lineMarkers
+
+class ElmRecursiveCallLineMarkerProviderTest : ElmLineMarkerProviderTestBase() {
```

Comment:
The tests should include some values and functions that are not recursive (to guard against false positives).